api/all/src/main/java/io/opentelemetry/api/common/AttributesBuilder.java

```diff
@@ -35,8 +36,8 @@ public interface AttributesBuilder {
   // version.
   <T> AttributesBuilder put(AttributeKey<Long> key, int value);
 
-  /** Puts a {@link AttributeKey} with associated value into this. */
-  <T> AttributesBuilder put(AttributeKey<T> key, T value);
+  /** Puts an {@link AttributeKey} with an associated value into this if the value is non-null. */
```

Comment:
I might even think an additional comment is useful -- something which conveys that any _existing_ value for that key will remain if null is passed (a null value doesn't remove/unset any current value).

Reply:
done.